JS.md

```diff
@@ -160,11 +160,19 @@ function, global, memory and table imports):
 * Let `v` be the value of performing [`Get`](http://tc39.github.io/ecma262/#sec-get-o-p)(`v`, `i.export_name`)
 * If `i` is a function import:
   * If `IsCallable(v)` is `false`, throw a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror).
```

Comment:
Can you make `IsCallable(v)` a link to the JavaScript standard, i.e. [`IsCallable(v)`](https://tc39.github.io/ecma262/#sec-iscallable)